website/source/docs/guides/kubernetes-reference.html.md

```diff
@@ -0,0 +1,184 @@
+--- 
+layout: "docs"
+page_title: "Kubernetes Consul Reference Architecture"
+sidebar_current: "docs-guides-k8s-reference-architecture"
+description: |-
+  This document provides recommended practices and a reference architecture. 
+---
+
+# Consul and Kubernetes Reference Architecture
+
+Preparing your Kubernetes cluster to successfully deploy and run Consul is an
+important first step in your production deployment process. In this guide you
+will prepare your Kubernetes cluster, independent of its underlying platform
+(AKS, EKS, GCP, etc). However, we will call out cloud specific differences when
+applicable. Before starting this guide you should have experience with
+Kubernetes, and have `kubectl` and helm configured locally. 
+
+By the end of this guide, you will be able to select the right resource limits
+for Consul pods, select the Consul datacenter design that meets your use case,
+and understand the minimum networking requirements. 
+
+## Infrastructure Requirements
+
+Consul server agents are responsible for the cluster state, responding to RPC
+queries, and processing all write operations. Since the Consul servers are
+highly active and are responsible for maintaining the cluster state, server
+sizing is critical for the overall performance, efficiency, and health of the
+Consul cluster. Review the [Consul Reference
+Architecture](/advanced/day-1-operations/reference-architecture#consul-servers)
+guide for sizing recommendations for small and large Consul datacenters. 
+
+The CPU and memory recommendations can be used when you select the resources
+limits for the Consul pods. The disk recommendations can also be used when
+selecting the resources limits and configuring persistent volumes. You will
+need to set both `limits` and `requests` in the Helm chart. Below is an example
+Helm chart for a Consul server in a large environment.
+
+```yaml 
+server 
+  resources: | 
+    requests: 
+      memory: "32Gi" 
+      cpu: "4" 
+      disk: "50Gi"
+   limits: 
```

Comment:
This and the following three lines seem to be missing a leading space (they should line up with `requests` above).